### PR TITLE
Adds datadog in orka

### PR DESCRIPTION
**Pull request by nikosd23**

### CODE REVIEW

File: src/initializers/datadog/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import requireInjected from '../../require-injected';
+import { getLogger } from '../log4js';
+
+let tracer;
+export default config => {
+  if (process.env.DD_SERVICE && process.env.DD_ENV) {
+    tracer = requireInjected('dd-trace').init();
+    tracer.use('koa', {
```
Comment (stavros-wb) [inline src/initializers/datadog/index.ts:8]: maybe we can have an optional config with an array of datadog plugins to be enabled?

**Review by stavros-wb — APPROVED** [review]

**Review by klesgidis — APPROVED** [review]
Nice!!
Maybe add the config option that enables datatog in readme (like newrelic)

File: src/initializers/datadog/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      blacklist: config?.datadog?.blacklistedPaths?.split(',') || ['/health']
```
Comment (nikostoulas) [inline src/initializers/datadog/index.ts:9]: 🔧 Initialise datadog.blacklistedPaths = ['/health'] in diamorphosis and then diamorphosis does the split.
```suggestion
      blacklist: config?.datadog?.blacklistedPaths
```
Maybe name it `dd.blacklistedPaths` to match DD_SERVICE and DD_ENV?
And in env: DD_BLACKLISTED_PATHS if it needs to change through an env?

**Review by jgerbe — APPROVED** [review]

**Review by nikostoulas — APPROVED** [review]

File: src/initializers/datadog/index.ts (hunk 2 above)
Comment (nikosd23) [reply src/initializers/datadog/index.ts:9]: I don't want to use DD_* as this convention is used by the dd-tracer itself.

File: src/initializers/datadog/index.ts (hunk 1 above)
Comment (nikosd23) [reply src/initializers/datadog/index.ts:8]: According to the API found [here](https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md)  the instrumentation of external libraries using the built-in plugins is done automatically. You can also configure disabled plugins through env variables which is the recommended way to configure datadog, that's why I kept the initialisation simple.

**Review by kzacharakis — APPROVED** [review]

**Pull request merged**